curriculum/challenges/english/20-upcoming-python/learn-lambda-functions-by-building-a-expense-tracker/657e895c6ab1b62db4b3438e.md

```diff
@@ -0,0 +1,99 @@
+---
+id: 657e895c6ab1b62db4b3438e
+title: Step 9
+challengeType: 20
+dashedName: step-9
+---
+
+# --description--
+
```

Comment:
This step is too long

Reply:
I have reduced this step, but I still tried to explain lambda functions as good as possible because that is the main focus for the project.